package.json

```diff
@@ -23,7 +23,6 @@
     "mqlight": "^1.0.0"
   },
   "devDependencies": {
-    "jscs": "^2.8.0",
     "eslint": "^1.10.3",
     "eslint-config-strongloop": "^1.0.2",
```

Comment:
I agree, this needs to happen to all IBM connectors. Will create some issues you can pull into planning (lots of work to do in this FA I would like to get on).